apps/api/docs/api-test-cases.md

```diff
@@ -42,7 +42,7 @@ Before running the test cases, make sure you have the following prerequisites in
    Alternatively, click the three dots beside the collection name and click `Run collection`
 
 3. Select the environment on top right.
-4. Click `Run osmo-notify` to execute the test cases.
+4. Click `Run osmoX` to execute the test cases.
```

Comment:
Update collection name(assuming it takes it from file name)

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b